### DISCUSSION TITLE
আমি একটি ড্রোন তৈরি করব

### DISCUSSION THREAD

**Discussion by Mdtorikullah** [iNavFlight/inav-configurator#2182] [General]
আমি কিভাবে ড্রোন তৈরি করতে পারি এর জন্য কি কি প্রয়োজন এবং কিভাবে প্রোগ্রাম কি হবে বিস্তারিত জানালে ভালো হয়

**Comment by mmosca** [comment]
Can you post your issue in english? I don't think anyone in the team speaks your language.
